abhayypatel.md

```diff
@@ -0,0 +1,45 @@
+
+The raspberry pi will play a key role in this system, serving as both a server to track cryptocurrency prices and a mail server to send out alerts. By hosting a mail server on the raspberry pi, the system can send out alerts to the user via email, ensuring that they receive timely notifications even when they are not physically near the alert system. In addition to email alerts, the system will also use text-to-speech technology to announce the cryptocurrency that is going up or down, providing a visual and auditory notification for the user.
+
+Overall, I am excited to build this project because it combines several different technologies and tools, including the raspberry pi, text-to-speech, and email alerts, to create a unique and useful alert system for cryptocurrency holders. The ability to stay informed about significant price movements in real-time can be valuable for anyone looking to make informed decisions about their cryptocurrency investments, and I am eager to see how this system can help people navigate the volatile world of cryptocurrency trading.
+
+
+## Plan
+
+1. Set up a Raspberry Pi and configure it as a server. This will involve installing the necessary operating system and setting up any required dependencies.
+
+2. Get a list of the top 1000 cryptocurrencies to monitor and use coinmarketcap to track their price real-time.
+
+3. Implement a system for detecting significant price changes in the cryptocurrency. This will involve setting up a threshold for the percentage change in price.
+
+4. Set up the Raspberry Pi to send email alerts when significant price changes are detected. This will involve installing and configuring a mail server on the Raspberry Pi, as well as writing code to send the emails when triggered by the price change detection system.
+
+5. Implement text-to-speech functionality to announce the cryptocurrency that is going up or down. I'd also be implementing the RBG and alarm sound to go off before the TTS announces the cryptocurrency.
+
+6. Test the system to ensure that it is working properly and sending alerts as expected. This will involve manually triggering alerts by simulating price changes and verifying that the alerts are sent and received correctly.
+
+7. Deploy the system and begin using it to stay informed about significant price changes in the cryptocurrency of choice. This will involve keeping the Raspberry Pi running and monitoring the alerts as they come in.
+
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+
```

Comment:
```suggestion
```

Reply:
> @abhayypatel this is really awesome! how do you plan on interacting with / programming the Raspberry Pi?

Hi there, thanks for the reply! I plan on interacting with the Raspberry Pi by also implementing an alert system which will play a loud alarm sound and flash RBG lights. The Raspberry Pi will also announce the signal through the speaker in TTS as stated earlier. And finally, the Raspberry Pi will manage the backend requests to validate a signal before it announces it. Please let me know if you’d like to know anything else.